pkgs/by-name/yo/your_spotify/client.nix

```diff
@@ -0,0 +1,54 @@
+{
+  apiEndpoint ? "localhost:8080",
+  fetchYarnDeps,
+  lib,
+  makeWrapper,
+  mkYarnPackage,
+  prefetch-yarn-deps,
+  src,
+  version,
+  yarn,
+}:
+mkYarnPackage rec {
+  inherit version src;
+  name = "your_spotify_client-${version}";
```

Comment:
The problem it that `mkYarnPackage` takes the name from the package.json located in the source folder. Due to the repository layout using npms workspaces this name is `your_spotify/root` which results in the package name generated being `your_spotify-root-1.8.0`. I wanted to avoid the root infix and thus set the name directly. But if the infix is less undesirable than using `name` directly I can change it of course.

Reply:
Then we can set name manually but we should definitely set pname+version

Reply:
I've set pname and version on both packages now.